contributing.md

```diff
@@ -26,4 +26,23 @@ That doesn't mean we don't want you to disagree with us! We just want to keep th
 
 ### On the use of pronouns
 
-Like a regular codebase, there's going to be some artifacts in the writing that we'll want to "refactor" over time. As the number of contributors increase, the usage of the pronoun "I" (which is still pretty common in the docs) might get confusing. Try to be mindful of that as you write!
+Like a regular codebase, there's going to be some artifacts in the writing that we'll want to "refactor" over time. As the number of contributors increase, the usage of the pronoun "I" (which is still pretty common in the docs) might get confusing. There's still some of that left, which we should remove over time. Add your name if you'd like to put your own story or opinion as advice.
+
+### Building the website
+
+Website generated by [Ronak Shah](https://ronakshah.net) with [mkDocs](http://www.mkdocs.org/)
+
+To generate the website (contents of `site/` directory) from source files (contents of `doc/` directory), first install mkDocs:
+
+```
+pip install --upgrade pip
+pip install mkdocs
+```
+
+and build the website:
+
+```
+mkdocs build
+```
+
+Don't forget to commit the `site/` directory too.
```

Comment:
Oh I didn't know about that command. Looks like `gh-deploy` also builds `site/` (my files have changed after running it). Looks like that's the right one to run.